docs/modules/ROOT/pages/reference/extensions/aws2-s3.adoc

```diff
@@ -44,3 +44,28 @@ Check the xref:user-guide/index.adoc[User guide] for more information about writ
 This extension auto-enables SSL support in native mode. Hence you do not need to add
 `quarkus.ssl.native=true` to your `application.properties` yourself. See also
 https://quarkus.io/guides/native-and-ssl[Quarkus SSL guide].
+
+== Additional Camel Quarkus configuration
+
+=== Optional integration with Quarkus Amazon S3
+
+If desired, it is possible to use the Quarkus Amazon S3 Client extension in conjunction with Camel Quarkus AWS2 S3.
+Note that this optional and not at all mandatory.
+Follow the https://quarkus.io/guides/amazon-s3#configuring-s3-clients[documentation] with some caveats.
+
+1. The Apache client type has to be selected by configuring the following property.
```

Comment:
FYI - I had to fix up the list continuation for aws2-ddb in https://github.com/apache/camel-quarkus/pull/3138. The same applies here.